pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -142,6 +142,10 @@ type reconciler struct {
 }
 
 func (rc *reconciler) Run(stopCh <-chan struct{}) {
+	if rc.kubeClient == nil {
+		klog.Warningf("kubeClient is nil, not entering the loop")
```

Comment:
I think people still use inline volumes in standalone mode (like emptyDir, hostpath). So I don't think we can completely turn off reconciler

Reply:
@msau42 
I will revert to the nil check if you don't have objection.

Reply:
So standalone mode only allows inline volumes, right?

Reply:
Correct, I think even inline volume types like secrets/configmaps may not work since those need API access as well.